src/commands/skullboard.py

```diff
@@ -0,0 +1,149 @@
+import os
+import requests
+import re
+from discord import Embed, Client
+from dotenv import load_dotenv
+from constants.colours import LIGHT_GREY
```

Comment:
```suggestion
import os
import requests
import re

from discord import Embed, Client
from dotenv import load_dotenv

from constants.colours import LIGHT_GREY
```

I suggest breaking imports up into clear blocks of standard, third party, and project-specific imports - as suggested in [PEP 8](https://peps.python.org/pep-0008/#imports).